api/controllers/auth_controller.js

```diff
@@ -2,9 +2,15 @@
 
 const config = require('../config/config');
 
-let jwt = require('jsonwebtoken');
-let bcrypt = require('bcrypt');
-let customer = require('../models/account_model');
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+const customer = require('../models/account_model');
+
+const jwt_options = {
+    expiresIn: "30d",
```

Comment:
30 days is pretty long... for a desktop application?  I think that we should maybe use this for testing mostly and not as a default feature

Reply:
Ahh yea, I'm not really sure what a good length would be. Do you have a suggestion for this value?

Reply:
no idea lol maybe 1 hour?

Reply:
I'm gonna make it a day just for ease of development, we can shorten it or add a "remember me for x days" or something like that down the road.